python/python-core/python-threading/threading-with-join.md

```diff
@@ -0,0 +1,189 @@
+---
+author: Stefan-Stojanovic
+
+tags:
+  - introduction
+
+type: normal
+
+category: must-know
+
+practiceQuestion:
+  formats:
+    - fill-in-the-gap
+    - type-in-the-gap
+  context: standalone
+revisionQuestion:
+  formats:
+    - fill-in-the-gap
+    - type-in-the-gap
+  context: standalone
+---
+
+# Threading With join()
+
+---
+
+## Content
+
+Previoulsy, we created a thread using `someFunction`[1]:
+
+```python
+firstThread = 
+  threading.Thread(
+    target=someFunction
+  )
+```
+
+Now, let's import the `time` module and use a different function to create one.
+
+> Using the `time` module we can see how `threading` works by implemeting the `time.sleep(NUM_OF_SECOND)` method.
```

Comment:
I think we should emphasize words using the *emphasized* or **bold** decoration, not code, such that we can differentiate between the two. 

```suggestion
> Using the `time` module we can see how `threading` works by implemeting the `time.sleep(NUM_OF_SECOND)` method.
```